sql/core/src/test/resources/sql-tests/inputs/join-empty-relation.sql

```diff
@@ -1,8 +1,3 @@
--- List of configuration the test suite is run against:
```

Comment:
This is to test the optimizer, don't need to run it with different join operators